### PR TITLE
Add tests and refactor trollstalker a bit

### PR DESCRIPTION
**Pull request by mraspaud**
This PR adds a simple functionality test for trollstalker.
At the same time the main functionality is moved out of the bin directory into the pytroll_collector package.

Another test is commented out for now, as this will be used in the next PR.

### CODE REVIEW

File: pytroll_collectors/trollstalker.py [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+    for key in config:
+        if 'var_' in key:
+            new_key = key.replace('var_', '')
+            var = config[key]
+            variables[new_key] = var
+    return variables
+
+
+def main(command_args=None):
+    """Run Trollstakler.
```
Comment (pnuu) [inline pytroll_collectors/trollstalker.py:292]:
```suggestion
    """Run Trollstalker.
```

**Review by pnuu — APPROVED** [review]
LGTM. Naturally lots of corner cases that are not covered by the simple test, but at least the main functionality now has a test. One typo fix inline which I'll commit, and then merge this to `main`
